lib/controller/page/type/page.summary/page.summary.controller.unit.spec.js

```diff
@@ -334,56 +334,6 @@ test('it does not attach json to submission when env var not present', async t =
   t.end()
 })
 
-test('TEAM PDF: it attaches PDF contents to email submission attachments', async t => {
-  getInstancePropertyStub.withArgs('service', 'emailTemplateTeam').returns('{firstname} submitted!')
-
-  const pageSummaryController = proxyquire('./page.summary.controller', {
-    '../../../../presenter/pdf-payload': {pdfPayload: pdfPayloadStub},
-    '../../../../middleware/routes-output/submission-data-with-labels': {submissionDataWithLabels: submissionDataWithLabelsStub},
-    '../../../../page/check-submits/check-submits': {checkSubmits: checkSubmitsStub},
-    '../../../../constants/constants': {
-      SERVICE_OUTPUT_EMAIL: 'bob@gov.uk'
-    }
-  })
-
-  const userdata = getUserDataMethods({input: {firstname: 'bob'}})
-
-  await pageSummaryController.postValidation({}, userdata)
-
-  const submissions = submitterClientSpy.getCall(0).args[0].actions
-  const emailEntries = submissions.filter(s => s.type === 'email')
-
-  t.equals(emailEntries[0].email_body, 'bob submitted!')
```

Comment:
this should stay the rest of the test can go

Reply:
Both agreed that this is tested below in another test